src/components/Customer.js

```diff
@@ -41,83 +84,97 @@ export class Customer extends Component {
   render() {
     const {
       customer,
```

Comment:
You have both a state for `customer` and a state for each of the properties in a customer (id, name, ..) is this neccesary?